civis/io/_tables.py

```diff
@@ -226,13 +248,25 @@ def f(x):
     if not outputs:
         raise EmptyResultError("Query {} returned no output."
                                .format(script_id))
+
     url = outputs[0]["path"]
+    file_id = outputs[0]["file_id"]
+    log.info('Exported results to Civis file %s (%s)',
+             outputs[0]["output_name"], file_id)
```

Comment:
i guess that spirit of logging as info was to make the file created more discoverable so that people could interact with the file more easily if they wanted to.

in order to do that they would have to submit this job as hidden=False, then find the script id, then run id, then look at run outputs.

Reply:
please confirm that debug is preferred

Reply:
Hm. I don't like the idea of making a file ID an "output" of this function. For a couple of reasons -- one is that it feels a bit like an implementation detail that the export becomes a Civis file, and we shouldn't commit to having this function always provide a file. The second is that printing something to the screen doesn't seem like a great way to surface information which we want the user to pick up and use somewhere else.

Given that Python offers very fine grained control of log outputs, I guess it's not that big of a deal. But I'd advocate for having all (or almost all) log emits from the client as debug level.

@keithing , do you have a preference here?

Reply:
updated to debug for now